### PR TITLE
move-to-abs

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR moves this repo to app-build-suite

### Checklist

- [ ] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: helm/dashboards/values.yaml [hunk 1]
```diff
@@ -3,10 +3,6 @@ global:
   namespace: monitoring
   serviceType: managed
 
-  project:
-    branch: "[[ .Branch ]]"
```
Comment (marieroque) [inline helm/dashboards/values.yaml:7]: You need to update the schema

**Review by marieroque — APPROVED** [review]

File: helm/dashboards/values.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/dashboards/values.yaml:7]: Thanks :)

File: helm/dashboards/Chart.yaml [hunk 2]
```diff
@@ -4,8 +4,7 @@ engine: gotpl
 home: https://github.com/giantswarm/dashboards
 icon: https://s.giantswarm.io/app-icons/grafana/1/light.svg
 name: dashboards
-appVersion: [[ .AppVersion ]]
```
Comment (QuantumEnigmaa) [inline helm/dashboards/Chart.yaml:7]: appVersion is entirely removed ? I guess it makes sense for dashboard though :thinking:

Comment (QuentinBisson) [reply helm/dashboards/Chart.yaml:7]: Yes we do not have an app version

**Pull request merged**